Gemfile

```diff
@@ -1,10 +1,10 @@
-# frozen_string_literal: true
-
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
```

Comment:
- [ ] Following the project requirements,  you should write Unit and Integration tests. You can use RSpec and Capybara testing libraries for that.